src/Model/Message.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\Model;
+
+use Symfony\Component\Validator\Constraints as Assert;
+
+class Message
+{
+    public const TYPE_SMS = 'sms';
+    public const TYPE_EMAIL = 'email';
+    /**
+     * @Assert\NotBlank(message="body cannot be empty", payload="101")
+     */
+    public string $body = '';
+
+    /**
+     * @Assert\NotBlank(message="type cannot be blank", payload="102")
+     */
+    public string $type = self::TYPE_EMAIL;
+
+    /**
+     * @return string
+     */
+    public function getBody(): string
+    {
+        return $this->body;
+    }
+
+    /**
+     * @param string $body
+     */
+    public function setBody(string $body): void
```

Comment:
Galime grąžinti ne void, o pačią Message klasę su self return tipu ir return $this statementu. Taip konstruojant objektą galime grąžiai nestinti setterius.